ListItemFormatter.xcodeproj/project.pbxproj

```diff
@@ -925,6 +925,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "$(inherited).ios";
 				SDKROOT = iphoneos;
 				SKIP_INSTALL = YES;
+				SWIFT_VERSION = 5.0;
```

Comment:
I think I configured things so that this could all be changed at the project level and not the target level so that we only have to adjust the value in one place